framework/wazuh/core/indexer/commands.py

```diff
@@ -1,13 +1,13 @@
 from dataclasses import asdict
-from typing import List, Optional
+from typing import List, Optional, Union
 
-from opensearchpy import NotFoundError
 from uuid6 import UUID
 
 from .base import BaseIndex, IndexerKey, remove_empty_values
-from wazuh.core.indexer.models.commands import Command, Status
 from wazuh.core.exception import WazuhResourceNotFound
+from wazuh.core.indexer.models.commands import Command, Result, Status
 
+ID_KEY = 'id'
 AGENT_ID_KEY = 'agent.id'
 STATUS_KEY = 'status'
```

Comment:
Also this one

Reply:
These keys refer to the `commands` index values fields and `IndexerKey` contains keys used by OpenSearch.

I would prefer to keep them separate because if we include every field name inside `IndexerKey` we will end up with hundreds of keys. But it's something that we can discuss.